lib/libc/string/wcscmp.c

```diff
@@ -51,8 +51,8 @@ wcscmp(const wchar_t *s1, const wchar_t *s2)
 {
 
 	while (*s1 == *s2++)
-		if (*s1++ == '\0')
+		if (*s1++ == L'\0')
 			return (0);
-	/* XXX assumes wchar_t = int */
```

Comment:
This still assumes wchar_t fits in an (unsigned) int, but I guess other places don't have an XXX so it's fine to drop...